moml/data/feature_transforms.py

```
@@ -0,0 +1,119 @@
+            data.x = torch.cat([data.x, padding], dim=1)
+        
+        return data
+class StandardizeTargets:
+    """A transform to standardize targets using pre-computed statistics."""
+    def __init__(self, stats_path="data/target_stats.yaml", dataset_name="qm9"):
+        try:
+            with open(stats_path, 'r') as f:
+                stats = yaml.safe_load(f)
+        except FileNotFoundError:
+            logging.error(f"Statistics file not found at {stats_path}")
+            raise FileNotFoundError(f"Statistics file not found: {stats_path}")
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)